kusto/conn.go

```diff
@@ -117,7 +117,7 @@ type execResp struct {
 	frameCh    chan frames.Frame
 }
 
-func (c *conn) execute(ctx context.Context, execType int, db string, query Stmt, payload string, properties requestProperties) (execResp, error) {
```

Comment:
Mind this is a breaking change despite it not being used.
Please comment on it in the commit.

Reply:
Is it a breaking change to change a private method?